components/button/index.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+
+const StyledButton = styled.button`
+  background-color: ${(props) => props.theme.colors[props.type]};
+
```

Comment:
Let's keep the CSS relatively compact and avoid blank line between properties. Let me know if you feel it's better otherwise.